src/otx/algo/instance_segmentation/heads/maskdino_encoder.py

```diff
@@ -307,7 +307,7 @@ def __init__(
         self.lateral_convs = lateral_convs[::-1]
         self.output_convs = output_convs[::-1]
 
-    def forward(self, features: dict[str, Tensor]) -> tuple[Tensor, Tensor, list[Tensor]]:
+    def forward_features(self, features: dict[str, Tensor]) -> tuple[Tensor, Tensor, list[Tensor]]:
```

Comment:
@kprokofi I don't think we should change this?

Reply:
I think it was merge conflicts due to the fact, that develop was refactored. Anyway, I reverted changes back for MaskDino Encoder and Decoder.